apps/total-typescript/src/video/module-lesson-list/large-screen-module-lesson-list.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import {LessonList} from './lesson-list'
+import ModuleLessonListHeader from './module-lesson-list-header'
+import {Module} from '@skillrecordings/skill-lesson/schemas/module'
+import {Section} from '@skillrecordings/skill-lesson/schemas/section'
+import {Lesson} from '@skillrecordings/skill-lesson/schemas/lesson'
+
+export const LargeScreenModuleLessonList: React.FC<{
+  exerciseResourcesRenderer: (
+    path: string,
+    module: Module,
+    lesson: Lesson,
+    section?: Section,
+  ) => void
+  module: Module
+  path: string
+}> = ({module, path, exerciseResourcesRenderer}) => {
```

Comment:
the renderer should have a default that displays minimum information (title?)

Reply:
there will always be a lesson title rendered: https://github.com/skillrecordings/products/blob/main/apps/protailwind/src/video/module-lesson-list/lesson-list.tsx#L353
or am I missing something?